tests/data_sources/gsp/test_gsp_model.py

```diff
@@ -47,8 +46,7 @@ def test_gsp_validation(configuration):  # noqa: D103
     GSP.model_validation(gsp)
 
     gsp.power_mw[0, 0] = np.nan
-    with pytest.raises(Exception):
-        GSP.model_validation(gsp)
+    GSP.model_validation(gsp)
```

Comment:
how come this does make an error now?

Reply:
Currently because the validation fills in the NaN's with zeros